Student/Student/Submissions/SubmissionDetails/SubmissionDetailsViewController.swift

```diff
@@ -65,6 +65,12 @@ class SubmissionDetailsViewController: UIViewController, SubmissionDetailsViewPr
         presenter?.viewDidDisappear()
     }
 
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        drawerContentViewController?.view.accessibilityElementsHidden = drawer?.height == 0
+        contentView?.accessibilityElementsHidden = drawer?.height != 0
+    }
```

Comment:
does this method execute each time the drawer state changes?

Reply:
Yes, thanks to the drawer calling `superview?.layoutIfNeeded()` here:

https://github.com/instructure/canvas-ios/blob/c2c6120d1c8320d20d5535a243123b0cb696ad6b/Student/Student/Submissions/SubmissionDetails/Views/Drawer.swift#L158